pallets/qf-polkavm/src/lib.rs

```diff
@@ -405,25 +414,31 @@ pub mod pallet {
 					}
 					0
 				},
-				|m: Vec<u8>| -> u64 {
-					sp_runtime::print(&*m);
+				print: |m: Vec<u8>| -> u64 {
+					let msg = alloc::string::String::from_utf8_lossy(&m);
+					let msg_log = alloc::format!("polkavm: {msg}");
+					sp_runtime::print(msg_log.as_str());
```

Comment:
log::{debug, info} has target / prefix and env_logger filtering capabilities (enable / disable depending on env variable), please consider it for the next version.